packages/components/accordion/accordion.js

```diff
@@ -0,0 +1,114 @@
+// Accordion component
+
+'use strict';
+
+module.exports = function Accordion() {
+    var accordion = {};
+
+    /**
+     * Accordion is build on top of the aria attributes by the w3c example:
+     *     https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html
+     *
+     * @example
+     * <div id="accordion">
+     *    <section class="collapse">
+     *        <h4 id="accordion-title-1">
+     *            <button aria-expanded="false" aria-controls="accordion-body-1">
+     *                Header 1
+     *            </button>
+     *        </h4>
+     *
+     *        <div id="accordion-body-1" aria-hidden="true" aria-labelledby="accordion-button-1">
+     *            Content 1
+     *        </div>
+     *    </section>
+     *
+     *    <section class="collapse">
+     *        <h4 id="accordion-title-2">
+     *            <button aria-expanded="false" aria-controls="accordion-body-2">
+     *                Header 2
+     *            </button>
+     *        </h4>
+     *
+     *        <div id="accordion-body-2" aria-hidden="true" aria-labelledby="accordion-title-2">
+     *            Content 2
+     *        </div>
+     *    </section>
+     * </div>
+     *
+     * import Accordion from '@sulu/web/packages/components/accordion/accordion';
+     * var component = new Accordion();
+     * component.initialize(document.getElementById('accordion'), {});
+     *
+     * @param {HTMLElement} el
+     * @param {object} options
+     */
+    accordion.initialize = function initialize(el, options) {
+        var i;
+        var button;
+
+        accordion.el = el;
+        accordion.items = [];
+
+        for (i = 0; i < el.children.length; i++) {
```

Comment:
```suggestion
        for (var i = 0; i < el.children.length; i++) {
```

Then you can remove the `var i;` above

Reply:
We have a eslint rule which prevents this because of how var behaves in browsers.

Reply:
okay then